pkgs/common-updater/git-updater.nix

```diff
@@ -3,9 +3,9 @@
 , common-updater-scripts
 }:
 
-{ pname
-, version
-, attrPath ? pname
+{ pname ? ""
```

Comment:
There are still empty strings as nulls in other files.

Reply:
Done.